README.md

```diff
@@ -52,6 +52,7 @@ The gem is available as open source under the terms of the [MIT License](https:/
 * Lockstep::MagicLink
 * Lockstep::Note
 * Lockstep::Payments
+* Lockstep::PaymentApplied
```

Comment:
can you add the other model added as well ServiceFabricPayment

Reply:
It is already there Lockstep::ServiceFabricPayment